crates/shared/src/consts.rs

```diff
@@ -1,3 +1,4 @@
 pub const EXPECTED_RPC_VERSION: &str = "0.7.0";
 pub const RPC_URL_VERSION: &str = "v0_7";
 pub const SNFORGE_TEST_FILTER: &str = "SNFORGE_TEST_FILTER";
+pub const FREE_RPC_PROVIDER_URL: &str = "https://free-rpc.nethermind.io/sepolia-juno/v0_7";
```

Comment:
nit: Should be removed completely for now

Reply:
Yes, it was removed as a default from the CastConfig::Default, but I would like to leave it in the commented template.